lib/minfraud/http_service/response.rb

```diff
@@ -7,24 +7,25 @@
 
 module Minfraud
   module HTTPService
-    # Response class for HTTP requests
+    # Response class for HTTP requests.
     class Response
-      # @attribute status
-      # @return [Integer] HTTP response status
+      # HTTP response status.
+      #
+      # @return [Integer, nil]
       attr_reader :status
 
-      # @attribute body
+      # HTTP response.
+      #
       # @return [Minfraud::Model::Score, Minfraud::Model::Insights,
```

Comment:
When will this be `nil`?

Reply:
It looks like it'll happen when there's a non-HTTP 200 response that isn't JSON for some reason. Though in practice we'll raise an error shortly after that happens.